monitoring/alerting/rules/other_alerts.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: 1
+groups:
+  - interval: 5m
+    name: SAS Viya Alerts
+    folder: Other Alerts
+    orgId: 1
+    rules:
+      - title: Crunchy Backrest Repo
+        annotations:
+          description:
+            "Checks to see /pgbackrest/repo1 filesystem is more than 50% full.
+
+            Go to the  URL to follow the troubleshooting steps."
+          summary:
```

Comment:
Remove quotes in DESCRIPTION.  Remove reference to URL.  Why is this in Other Alerts instead of Database Alerts with the other Crunchy-related Alerts?